### PR TITLE
Kafka datadog: clarify Kafka metrics vs Kafka consumer metrics

### PR DESCRIPTION
**Pull request by juha-aiven**
# What

Split the chapter of customising metrics into two - first for the main Kafka integration and the other for the  Consumer Integration.

Add an example how to use exclude_topics.

# Why

The issue was that documentation failed to mention that the four keys include_topics, exclude_topics, include_consumer_groups and exclude_consumer_groups only affect the consumer integration. And that kafka_custom_metrics applies to Kafka integration.

### CODE REVIEW

File: docs/products/kafka/howto/datadog-customised-metrics.rst [hunk 1]
```diff
@@ -42,26 +42,44 @@ Before customising the metrics, make sure that you have a Datadog endpoint confi
 To customise the metrics sent to Datadog, you can use the ``service integration-update`` passing the following customised parameters:
 
 * ``kafka_custom_metrics``: defining the comma separated list of custom metrics to include (within ``kafka.log.log_size``, ``kafka.log.log_start_offset`` and ``kafka.log.log_end_offset``)
+
+As example to sent the ``kafka.log.log_size`` and ``kafka.log.log_end_offset`` metrics execute the following code:
```
Comment (harshini-rangaswamy) [inline docs/products/kafka/howto/datadog-customised-metrics.rst:46]:
```suggestion
For example, to send the ``kafka.log.log_size`` and ``kafka.log.log_end_offset`` metrics, execute the folllowing code:
```

File: docs/products/kafka/howto/datadog-customised-metrics.rst [hunk 2]
```diff
@@ -42,26 +42,44 @@ Before customising the metrics, make sure that you have a Datadog endpoint confi
...
+    avn service integration-update                                                \
+        -c kafka_custom_metrics=['kafka.log.log_size','kafka.log.log_end_offset'] \
+        INTEGRATION_ID
+
+Once the update is successful and metrics have been collected and pushed, you should see them in your Datadog explorer.
+
+.. seealso:: Learn more about :doc:`/docs/integrations/datadog`.
+
+
+Customise Apache Kafka® Consumer Integration metrics sent to Datadog
```
Comment (harshini-rangaswamy) [inline docs/products/kafka/howto/datadog-customised-metrics.rst:59]: Do you think we can shorten the heading a bit, say something like: 

Customize  Apache Kafka® consumer metrics for Datadog
-----------------------------------------------------------

File: docs/products/kafka/howto/datadog-customised-metrics.rst [hunk 3]
```diff
@@ -42,26 +42,44 @@ Before customising the metrics, make sure that you have a Datadog endpoint confi
...
+====================================================================
+
+`Kafka Consumer Integration <https://docs.datadoghq.com/integrations/kafka/?tab=host#kafka-consumer-integration>`_ collects metrics for message offsets.
+
+To customise the metrics sent from this Datadog integration to Datadog, you can use the ``service integration-update`` passing the following customised parameters:
+
 * ``include_topics``: defining the comma separated list of topics to include
 
-.. Tip:: 
+.. Tip::
```
Comment (harshini-rangaswamy) [inline docs/products/kafka/howto/datadog-customised-metrics.rst:68]:
```suggestion
  .. Note::
```

Comment (harshini-rangaswamy) [reply docs/products/kafka/howto/datadog-customised-metrics.rst:68]: I think "Note" is better than a "Tip" here.

File: docs/products/kafka/howto/datadog-customised-metrics.rst [hunk 4]
```diff
@@ -42,26 +42,44 @@ Before customising the metrics, make sure that you have a Datadog endpoint confi
...
 
     By default, all topics are included.
 
 * ``exclude_topics``: defining the comma separated list of topics to exclude
 * ``include_consumer_groups``: defining the comma separated list of consumer groups to include
 * ``exclude_consumer_groups``: defining the comma separated list of consumer groups to include
 
-
-As example to sent the ``kafka.log.log_size`` and ``kafka.log.log_end_offset`` metrics for ``topic1`` and ``topic2`` execute the following code:
+As example to include topics ``topic1`` and ``topic2`` and exclude topic ``topic3`` execute the following code:
```
Comment (harshini-rangaswamy) [inline docs/products/kafka/howto/datadog-customised-metrics.rst:76]:
```suggestion
For example, to include topics ``topic1`` and ``topic2``, and exclude ``topic3``, execute the following code:
```

**Review by harshini-rangaswamy — CHANGES_REQUESTED** [review]
Added some minor suggestions.

**Comment by juha-aiven** [comment]
Could you @harshini-rangaswamy apply the changes directly in the branch? As long as they are this kind of formatting, spelling etc related changes, I'm 100% fine with those. The main thing that the PR does is to split the page in to the chapters and if that's fine for you, I'm sure you figure out a better layout and formatting than me. 🙇🏻

**Review by harshini-rangaswamy — APPROVED** [review]

**Review by AnatolyPopov — APPROVED** [review]

**Pull request merged**